Netezos.Tests/Utils/Extensions/BytesExtension.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+
+namespace Netezos.Tests
+{
+    static class BytesExtension
```

Comment:
I copy/pasted the `BytesExtension` for temporary support while writing these methods. How do we want to handle this support? Should I make the `BytesExtension` public in Netezos and use it from here? Some common lib for the tools? Just copy/paste the required methods? Some other approach? Let me know your thoughts.

Reply:
I think you don't really need them. I suggest you use `ReadOnlySpan<T>` or `ReadOnlyMemory<T>` to avoid excessive memory allocation. Or at least look at `MemoryStream` with `BinaryReader`.

Reply:
Sounds great. I was considering `ReadOnlySpan<T>` and `ReadOnlyMemory<T>` but hadn't yet seen it used elsewhere so I wasn't sure if it would be acceptable. I'm glad to use it!